.circleci/config.yml

```diff
@@ -50,6 +50,7 @@ jobs:
       - run:
           name: "Run Python Tests"
           command: |
+            pytest --cache-clear demo/tests/
```

Comment:
Going forward I want to include the demo pipeline into the tests suite for `dynamicio`. It's a fast an reliable way to run "integration" tests. Also, making sure that the `demo` does not break will lead to better docs (as the demo is only used in the docs)